docs/source/javascripts/site.js

```diff
@@ -105,7 +261,8 @@ window.modules.search = (function () {
         bindToSearchInput = function (lunr) {
             $('#lunr-search').on('keyup', function (event) {
                 var results = lunr.index.search(event.target.value);
-                display(results, lunr);
+                var plugin_result = pluginResult(event.target.value);
```

Comment:
Don't use `snake_case` in JS, prefer `camelCase`. ESLint must not be checking this dir, we should fix that at some point.